components/dashboard/src/user-settings/SelectIDE.tsx

```diff
@@ -74,149 +71,69 @@ export default function SelectIDE(props: SelectIDEProps) {
-                    <CheckBox
-                        title="Latest Release (Unstable)"
-                        desc={
-                            <span>
-                                Use the latest version for each editor.{" "}
-                                <a
-                                    className="gp-link"
-                                    target="_blank"
-                                    href="https://code.visualstudio.com/blogs/2016/02/01/introducing_insiders_build"
-                                    rel="noreferrer"
-                                >
-                                    Insiders
-                                </a>{" "}
-                                for VS Code,{" "}
-                                <a
-                                    className="gp-link"
-                                    target="_blank"
-                                    href="https://www.jetbrains.com/resources/eap/"
-                                    rel="noreferrer"
-                                >
-                                    EAP
-                                </a>{" "}
-                                for JetBrains IDEs.
-                            </span>
-                        }
-                        checked={useLatestVersion}
-                        onChange={(e) => actuallySetUseLatestVersion(e.target.checked)}
-                    />
-                </>
-            )}
-        </>
-    );
-}
-
-function orderedIdeOptions(ideOptions: IDEOptions) {
-    // TODO: Maybe convert orderKey to number before sort?
-    return Object.entries(ideOptions.options)
-        .filter(([_, x]) => !x.hidden)
-        .sort((a, b) => {
-            const keyA = a[1].orderKey || a[0];
-            const keyB = b[1].orderKey || b[0];
-            return keyA.localeCompare(keyB);
-        });
-}
-
-function renderIdeOption(
-    option: IDEOption,
-    selected: boolean,
-    version: IDEOption["imageVersion"],
-    onSelect: () => void,
-): JSX.Element {
-    const shouldShowOptionType = option.type !== "desktop" || option.title === "VS Code"; // Force show of "Desktop" in the list for VS Code Desktop
-    const card = (
-        <SelectableCardSolid className="w-36 h-44" title={option.title} selected={selected} onClick={onSelect}>
-            {version ? (
-                <span
-                    className={`font-normal font-mono text-xs ${
-                        selected
-                            ? "text-gray-100 dark:text-gray-600"
-                            : "text-gray-400 contrast-more:text-gray-600 dark:text-gray-500"
-                    } pl-1 self-start`}
-                    title="The IDE's current version on Gitpod"
-                >
-                    {version}
-                </span>
-            ) : (
-                <span
-                    style={{
-                        minHeight: "1rem",
+            <div className="w-112 my-4">
+                <SelectIDEComponent
```

Comment:
**issue**: Probably not the best line of code to comment on this, but could we make sure the editor dropdown renders above the modal (`z-index`)? This is an issue regardless if we change the modal width in the scope of this PR.

| Issue on current modal width | Issue on changed modal width |
|-|-|
| <img width="1512" alt="Screenshot 2023-03-16 at 13 59 56" src="https://user-images.githubusercontent.com/120486/225610324-60bb311e-46c1-45f2-bc38-82fdc3d20401.png"> | <img width="1512" alt="Screenshot 2023-03-16 at 13 54 39" src="https://user-images.githubusercontent.com/120486/225610170-43afe068-db0f-4e16-817a-eede9d7ca734.png"> |

Reply:
@gtsiolis how did you trigger the modal? I can't seem to be able to get to it from the UI

Reply:
@filiptronicek We show this modal to all new users—you can reproduce if you delete your account and try to login again. 💡

Reply:
@gtsiolis I think I got it a couple of times, but it's always under the new onboarding experience and when I finish that, it's gone.

Reply:
@filiptronicek The new onboarding flow is not available on preview environments, see [relevant discussion](https://gitpod.slack.com/archives/C04PH56LZK5/p1678105071868449) (internal).

❓ Are you seeing the new onboarding flow in the preview environment for this PR? 💭 

💡 I could reliably re-trigger this modal after deleting my account quite a few times in this preview environment. Steps top reproduce:
1. Delete the account from [`/user/account`](https://filiptroni9852fbf578.preview.gitpod-dev.com/user/account) in the preview environment.
2. Try to logging in again.

Reply:
> ❓ Are you seeing the new onboarding flow in the preview environment for this PR? 💭 

Yes

Reply:
That's strange—could you share a screenshot of what you're seeing and what URL you're using?

Reply:
Aaah 🤦, I was explicitly mention in the feature flag to be included. Thanks for the help.

Reply:
Resolving this as the modal width now seems better! 📕

Reply:
I have played around with the positioning of the dropdown. I have tried many different tricks with positioning and setting the `z-index`, but only one of those attempts actually worked out.

We apparently need to use `position: fixed` for the position (currently, `position: absolute` is used), which is problematic because of its impact of relative positioning -> when positioning a `fixed` element, I could not get the dropdown options to be relative to the dropdown component, but that should be intended `fixed` behavior [[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/position#values)]. The problem is that now the dropdown options are flying on the top of the viewport and the only way to get them to their proper position is to manually try out different `top` values, until one is seemingly correct[^1].

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/29888641/225723688-041757d5-84f0-4e6d-b85b-ced80b0fa30b.png">

[^1]: which is obviously problematic, because it depends on the height of the content above the dropdown, so that if you change the "Choose the editor for opening workspaces" text and make it a bit longer, the `top` value will need to be re-adjusted.

Reply:
cc @selfcontained do you know of anything that might help here?

Reply:
@filiptronicek Here's a somewhat radical idea that could help: Last time I checked there was consensus to drop this editor selection and replace it with the new onboarding flow. Given that we're already doing this on Gitpod.io, we could drop this all together which can help bypass this issue. Later on, we could think about reusing the onboarding for Dedicated, which already includes this editor selection in the step 2. 

@loujaybee What do you think? Dropping this could help as it's only our team who sees this now in the preview environments. 🏓

Reply:
@filiptronicek We've got green flag to remove this, see [relevant discussion](https://gitpod.slack.com/archives/C01KGM9BH54/p1678996938846779?thread_ts=1678995547.356729&cid=C01KGM9BH54) (internal). Cc @loujaybee 

Could you make this change in this PR? Otherwise, a separate PR sounds also fine—your call. 🏓